kani-driver/src/cbmc_output_parser.rs

```diff
@@ -549,12 +550,12 @@ pub fn process_cbmc_output(
     mut process: Child,
     extra_ptr_checks: bool,
     output_format: &OutputFormat,
-    output_filename_opt: Option<&Path>,
-) -> VerificationStatus {
+) -> VerificationOutput {
     let stdout = process.stdout.as_mut().unwrap();
     let mut stdout_reader = BufReader::new(stdout);
-    let parser = Parser::new(&mut stdout_reader, output_filename_opt);
+    let parser = Parser::new(&mut stdout_reader);
     let mut result = false;
+    let mut processed_items: Vec<ParserItem> = Vec::new();
 
     for item in parser {
```

Comment:
nit: you could change this to be something like 
```rust
let processed_items = parser.filter_map(|item| {
  if item.must_be_skipped() {
    None;
  }
  // ...
}
```

Reply:
Great suggestion, thanks!